.github/workflows/build.yml

```
@@ -98,3 +98,55 @@ jobs:
       - name: Setup tmate debug session
         uses: mxschmitt/action-tmate@v3
         if: env.RUN_TMATE
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version:
+          - "1.18"
+          - "1.19"
+    steps:
+      # We need the Go version and Go cache location for the actions/cache step,
```

It looks like this got accidentally removed in https://github.com/cisagov/con-pca-tasks/commit/7751db53ea6145e573bea708aea853b28cc8aea0 and was the source of test failures.

```suggestion
      - uses: actions/checkout@v3
      # We need the Go version and Go cache location for the actions/cache step,
```

---

resolved with [5b1493b](https://github.com/cisagov/con-pca-tasks/pull/1/commits/5b1493b033102587c6a7655d569ca0ce321f8f8f)

---

Would you please move the `actions/checkout` step above the comment that is intended for the `actions/setup-go` step?

---

@mcdonnnj resolved with [98209ca](https://github.com/cisagov/con-pca-tasks/pull/1/commits/98209ca2c2f00135c5a455a5fdc8fc71f9493b51)